### ISSUE TITLE
the loss value 

### ISSUE THREAD

**Issue by wenouyang** [loliverhennigh/ultrasound-nerve-segmentation-in-tensorflow#1]
Hi Oliver,

Thanks for sharing the code.

I experiment the code with the ultrasound-nerve-segmentation data set, and found that even even after 100K steps, the loss value is still very high, For instance,

loss value is:  1.18567e+06  
step is:  101428   

Would you like to elaborate any insight for this?  Thanks a lot.

**Comment by loliverhennigh** [comment]
Hey,

Sorry I never responded to this. I changed a bunch of stuff with the model and loss function. Now the loss goes down and it gets about .61230 on the kaggle eval set. 

Thanks for the interest!
